graph/flow/control_flow_bench_test.go

```diff
@@ -31,6 +32,9 @@ func BenchmarkDominators(b *testing.B) {
 
 	fis, err := ioutil.ReadDir(testdata)
 	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
just thinking out loud, but isn't now the recommended way to use `errors.Is` or `errors.As`?

Reply:
That seems heavy for this.